pages/common/resume.md

```diff
@@ -0,0 +1,20 @@
+# resume
+
+> CLI tool to easily setup a new resume.
+> More information: <https://github.com/jsonresume/resume-cli>.
+
+- Create a new `resume.json` file in the current working directory:
+
+`resume init`
+
+- Validate a `resume.json` against schema tests to ensure it complies with the standard:
+
+`resume validate`
+
+- Export a resume locally in a stylized HTML or PDF format:
+
+`resume export {{path/to/output}}`
```

Comment:
Does the format depend on the file extention?
If so, `.../output.pdf|.html` looks weird, but maybe `path/to/html_or_pdf`?

Reply:
Sounds fair, I think that could be nice.

```suggestion
`resume export {{path/to/html_or_pdf}}`
```